pkgs/applications/misc/keystore-explorer/default.nix

```diff
@@ -0,0 +1,43 @@
+{ fetchzip, stdenv, jdk8, runtimeShell }:
+
+let
+  kseJdk = jdk8;
+in
+stdenv.mkDerivation rec {
+  version = "5.4.4";
+  pname = "keystore-explorer";
+  src = fetchzip {
+    url = "https://github.com/kaikramer/keystore-explorer/releases/download/v${version}/kse-544.zip";
+    sha256 = "01kpa8g6p6vcqq9y70w5bm8jbw4kp55pbywj2zrhgjibrhgjqi0b";
+  };
+
+  buildInputs = [ kseJdk ];
```

Comment:
Technically you don't even need it here as it's going to get referenced properly in `$out/bin/` anyway.